src/commands/list/list.ts

```diff
@@ -58,14 +58,15 @@ export const commandList = command({
 
 export async function* asyncFilter<T extends { path: string }>(
   source: AsyncGenerator<T>,
-  filter: string,
+  opts?: { include?: string; exclude?: string }, //filter: string,
```

Comment:
Is `//filter: string,` intentionally being left?

Reply:
Nope!